arangod/Cluster/ClusterMethods.cpp

```diff
@@ -199,13 +199,14 @@ OperationResult handleResponsesFromAllShards(
     std::vector<futures::Try<arangodb::network::Response>>& responses,
     std::function<void(Result&, VPackBuilder&, ShardID&, VPackSlice)> handler,
     std::function<void(Result&, VPackBuilder&)> pre = [](Result&, VPackBuilder&) -> void {},
-    std::function<void(Result&, VPackBuilder&)> post = [](Result&, VPackBuilder&) -> void {}) {
+    std::function<void(Result&, VPackBuilder&)> post = [](Result&, VPackBuilder&) -> void {},
+    OperationOptions const options = OperationOptions()) {
```

Comment:
Is this intentionall passed by value? I also would prefer to not use default function parameters, as it makes it easy to overlook necessary changes at call sites.
```suggestion
    OperationOptions const& options) {
```